plugin/al/plugin/AL_USDMayaTestPlugin/test_translators_NurbsCurveTranslator.cpp

```diff
@@ -66,15 +66,15 @@ MObject createNurbStage(bool useSingleWidth=false)
   VtVec3fArray points = VtVec3fArray(5);
   for(uint32_t i = 0 ; i< pointsa.size(); ++i)
   {
-    memcpy(&points[i], &pointsa[i], 3*sizeof(float));
+    memcpy((void*)&points[i], &pointsa[i], 3*sizeof(float));
```

Comment:
Same suggestion as above, but here I think I'd also remove the loops and include the array size in the range being copied?

Reply:
Yes, I think we can skip the intermediates and the copy as well.